server/engine/src/main/java/org/eclipse/lsp/cobol/service/delegates/completions/CompletionStorage.java

```diff
@@ -95,9 +92,10 @@ private void fillInStorage(Map<String, T> props) {
 
   private void setDialects(List<Object> dialectObject) {
     JsonArray jsonObj = (JsonArray) dialectObject.get(0);
-    this.dialectList = Streams.stream(jsonObj).map(JsonElement::getAsString).collect(toList());
+    List<String> dialectList = Streams.stream(jsonObj).map(JsonElement::getAsString).collect(toList());
     if (!dialectList.isEmpty())
       this.dialectType = dialectList.contains(DaCoDialect.NAME) ? DaCoDialect.NAME : IdmsDialect.NAME;
+    dialectList.add(CICS_TRANSLATOR_DIALECT);
```

Comment:
Looks like this list is not used anywhere.

Reply:
Updated. It is not needed